execution/evm/abi/core.go

```diff
@@ -41,7 +41,7 @@ func ReadAbiFormulateCallFile(abiLocation, binPath, funcName string, args []stri
 	return Packer(abiSpecBytes, funcName, args...)
 }
 
-func ReadAbiFormulateCall(abiSpecBytes []byte, funcName string, args []string) ([]byte, error) {
+func ReadAbiFormulateCall(abiSpecBytes []byte, funcName string, args []string) ([]byte, bool, error) {
```

Comment:
doc comment

Reply:
Add godoc and give function sensible name